containersQa/testlib.bash

```diff
@@ -129,28 +130,23 @@ function skipIfRhel7OsExecution() {
 # See https://github.com/containers/podman/issues/19556
 #
 function prepareUserSession() {
-  if [[ -z $USERNAME ]] ; then
-    echo "prepareUserSession: No username specified"
-    return
-  fi
-  echo "prepareUserSession for user \"$USERNAME\""
-  loginctl enable-linger $USERNAME 
-  if [[ $? -ne 0 ]]; then
-    echo "prepareUserSession: FAILED. loginctl enable-linger exit status $?"
-    return
+  if [[ -z $PD_PROVIDER ]]; then
+    echo "prepareUserSession: No PD_PROVIDER set"
+  else
+    echo "$PD_PROVIDER version:"
+    $PD_PROVIDER --version
   fi
+  echo "prepareUserSession for user \"$USER\""
+  loginctl enable-linger $USER || echo "prepareUserSession: FAILED"
```

Comment:
Note that we use `$USER` (the user running podman) and not `$USERNAME` (the user inside the container, be it `default` or whatever).